flink-tests/src/test/java/org/apache/flink/test/query/AbstractQueryableStateITCase.java

```diff
@@ -534,33 +537,66 @@ public Integer getKey(Tuple2<Integer, Long> value) throws Exception {
 	 * Retry a query for state for keys between 0 and {@link #NUM_SLOTS} until
 	 * <tt>expected</tt> equals the value of the result tuple's second field.
 	 */
-	private void executeValueQuery(final Deadline deadline,
-		final QueryableStateClient client, final JobID jobId,
-		final QueryableStateStream<Integer, Tuple2<Integer, Long>> queryableState,
-		final long expected) throws Exception {
+	private void executeValueQuery(
```

Comment:
Yes you are right! Value here is used in the sense of "asking the value for a key" not in the sense of `ValueState` which is a scalar. I will change this.